corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -239,7 +242,9 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
     <span class="ix" data-bind="text: ixInfo($data)"></span>
     <div class="eoq"></div>
   </div>
+  {% comment %} is this warning redundant/vestigial? {% endcomment %}
```

Comment:
It looks like it's meant to make sure that errors about questions being required but not answered are shown when the user tries to submit a form, and even if the unanswered questions are hidden because they're inside collapsed groups, they should still be shown. Does that sound right? If you remove this code, does that workflow still work?

Reply:
Yes, that sounds right. That's exactly what happens. That comment was a note for me to come back to and delete once I'd checked on it. I just missed removing it.